api/src/main/resources/metadata/concepts_and_drugs/Drug.xml

```diff
@@ -580,6 +580,176 @@
     <drug drug_id="1068" concept_id="86007" name="Ursodeoxcholic acid 300mg" combination="0" dosage_form="1513" strength="300mg" creator="1" date_created="2024-02-10 10:34:54" retired="0" uuid="5afbbf7e-7d1e-463a-89ea-435879a56c66"/>
     <drug drug_id="1069" concept_id="279" name="Insulin Mixtard 30/70 100 IU/ml" combination="0" dosage_form="1513" strength="100 IU/ml" creator="1" date_created="2024-02-10 10:34:54" retired="0" uuid="9f500be3-68a8-4856-a628-88197af2e812"/>
     <drug drug_id="1070" concept_id="175331" name="Rilpivirine 25mg" combination="0" dosage_form="1513" strength="25mg" creator="1" date_created="2024-02-10 10:34:54" retired="0" uuid="39c0149d-6908-4c8c-a78e-e097378f5e8b"/>
+
+    <drug  drug_id="1071"  concept_id="199143"  dosage_form="165528"    name="BETA- N CREAM  15mg" combination="0"   creator="1"  date_created="2025-02-25 10:34:54"  retired="0"   uuid="053ca2de-e8d8-45fb-b35f-f2925f22a971"   strength="15mg" />
+    <drug  drug_id="1072"  concept_id="199144"  dosage_form="1513"    name="DAPAGLIFLOZIN  10mg" combination="0"   creator="1"  date_created="2025-02-25 10:34:54"  retired="0"   uuid="053ca2de-e8d8-45fb-b35f-f2925f22a972"   strength="10 MG" />
+    <drug  drug_id="1073"  concept_id="199145"  dosage_form="1513"    name="ACECCLOFENAC 100mg" combination="0"   creator="1"  date_created="2025-02-25 10:34:54"  retired="0"   uuid="053ca2de-e8d8-45fb-b35f-f2925f22a973"   strength="100 MG" />
+    <drug  drug_id="1074"  concept_id="89"  dosage_form="1513"    name="ACECCLOFENAC AND PARACETAMOL 100/50mg" combination="0"   creator="1"  date_created="2025-02-25 10:34:54"  retired="0"   uuid="053ca2de-e8d8-45fb-b35f-f2925f22a974"   strength="100/50 MG" />
```

Comment:
89 does not match this drug given that its a combination of ACECCLOFENAC AND PARACETAMOL you might want to create this concept

Reply:
@Mwanje see more comments